### PR TITLE
OSDOCS-3154: Adding clarification for choosing the correct OS and architecture

### PR DESCRIPTION
**Pull request by kelbrown20**
https://issues.redhat.com/browse/OSDOCS-3154
Applies to 4.10+

Preview: https://deploy-preview-40705--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-default.html#installation-obtaining-installer_installing-aws-default

### CODE REVIEW

File: modules/installation-obtaining-installer.adoc [hunk 1]
```diff
@@ -95,6 +95,11 @@ cluster, even if the cluster failed during installation. To remove your cluster,
 {product-title} uninstallation procedures for your specific cloud
 provider.
 ====
++
+[IMPORTANT]
+====
+Ensure that you download the OpenShift installer that corresponds with your host operating system and architecture.
```
Comment (codyhoag) [inline modules/installation-obtaining-installer.adoc:101]: I think the wording is good and specifying the architecture makes sense with the AWS ARM scenario. My only recommendation would be to possibly move this info into the step 3 wording. What about something like this?
```
3. Navigate to the page for your installation type, download the installation program that
corresponds to your host operating system and architecture, and place the file in the directory
where you will store the installation configuration files.
```
Stacking three important admonitions together isn't best practice. I think having the actual step description mention all requirements would be ideal. Feel free to wordsmith that suggestion to make it even more clear :slightly_smiling_face:

**Review by codyhoag — COMMENTED** [review]
Just a brief comment, but overall, I think making that distinction is important and a good idea.

File: modules/installation-obtaining-installer.adoc (hunk 1 above)
Comment (kelbrown20) [reply modules/installation-obtaining-installer.adoc:101]: Yeah! Thats a great idea, and makes more sense.

**Comment by ahardin-rh** [comment]
The latest updates LGTM!

**Comment by jeffdyoung** [comment]
👍 from Devel

**Pull request merged**